src/components/ConversionFileErrorModal/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button } from "antd";
+import React from "react";
+
+import CustomModal from "../CustomModal";
+import { Exclamation } from "../Icons";
+
+import styles from "./style.css";
+
+interface ConversionFileErrorModalProps {
+    closeModal: () => void;
+}
+
+const ConversionFileErrorModal: React.FC<ConversionFileErrorModalProps> = ({
+    closeModal,
+}) => {
+    const footerButton = (
+        <Button type="primary" onClick={closeModal}>
+            OK
+        </Button>
+    );
+
+    return (
+        <CustomModal
+            className={styles.errorModal}
+            title="File import cannot be completed"
+            open
+            footer={footerButton}
+            width={426}
+            centered
+            onCancel={closeModal}
+        >
+            <div>
+                <div className={styles.redText}>
+                    {" "}
+                    {Exclamation}{" "}
+                    {`We're sorry, there was a problem importing your file.`}
+                </div>{" "}
+                You may want to double check that the file you selected is a
+                Smoldyn file and try again. For further assistance, please visit
```

Comment:
the `smoldyn` here should be a variable based on what engine they chose

Reply:
Ok, fixed this and it led to some other changes. 

Instead of an array of file extensions we should have a map, and I defined a SET_CONVERSION_ENGINE block in the reducer. Actually working with conversion engines is out of scope for this PR, but this sets us up to be dynamic when we get to that issue.